### ISSUE TITLE
DB-Datetime Field error by creating new Form

### ISSUE THREAD

**Issue by colletda** [nextcloud/forms#180]
**Which Forms version are you running?** (see apps page)
1.1.0
**Nextcloud or ownCloud?:**
NextCloud
**Nextcloud/ownCloud version:** (see Nextcloud admin page)
18.0.0

Steps to reproduce
1 install nexcloud 18
2 install form
3 create new form
4 create new question
5 Expected behaviour
6 Shoudl save the form

Actual behaviour
Form can't be saved

Server configuration
Operating system:
Synology DSM 6 (Debian)
Web server:
Apache
Database:
MariaDB 10
PHP version:
7.3

Updated from an older Nextcloud or fresh install:
new installation

Signing status:

Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
No errors have been found.

**List of activated apps:**
Teamfolder
Talk
Mail
Contacts
Calendar
Deck
Tasks
Accounsments
Forms
News

**Nextcloud configuration:**

If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your instance's installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)

**Are you using external storage, if yes which one:** local/smb/sftp/...
local
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
Chrome
**Operating system:**
Win10
**CalDAV-clients:**
none
#### Web server error log
Insert your webserver log here

#### Nextcloud log (data/nextcloud.log)
Doctrine\DBAL\Exception\DriverException: An exception occurred while executing 'INSERT INTO oc_forms_events (title, description, is_anonymous, unique, access, owner, created, hash) VALUES(?, ?, ?, ?, ?, ?, ?, ?)' with params ["Test2", "", 0, false, "public", "Daniel", "2020-01-22 06:53:51", "8c06ZvbRLs9q9R28"]: SQLSTATE[22007]: Invalid datetime format: 1366 Incorrect integer value: '' for column nextcloud.oc_forms_events.unique at row 1

#### Browser log

**Comment by MAGICCC** [comment]
Yup I do have the same problem. I thought first its some locale issue, but seems not
Here is also my table structure:
```
+----------------+---------------+------+-----+---------+----------------+
| Field          | Type          | Null | Key | Default | Extra          |
+----------------+---------------+------+-----+---------+----------------+
| id             | int(11)       | NO   | PRI | NULL    | auto_increment |
| hash           | varchar(64)   | YES  |     | NULL    |                |
| title          | varchar(128)  | NO   |     | NULL    |                |
| description    | varchar(1024) | NO   |     | NULL    |                |
| owner          | varchar(64)   | NO   |     | NULL    |                |
| created        | datetime      | YES  |     | NULL    |                |
| access         | varchar(1024) | YES  |     | NULL    |                |
| expire         | datetime      | YES  |     | NULL    |                |
| is_anonymous   | int(11)       | YES  |     | 0       |                |
| full_anonymous | int(11)       | YES  |     | 0       |                |
| unique         | int(11)       | YES  |     | 0       |                |
+----------------+---------------+------+-----+---------+----------------+
```

**Comment by nightbluepn** [comment]
https://github.com/nextcloud/forms/issues/176#issuecomment-581546056

**Comment by jotoeri** [comment]
Is it solved? One error got fixed by v1.1.1. Or do you still have another issue?

**Comment by jotoeri** [comment]
>  Incorrect integer value: '' for column nextcloud.oc_forms_events.unique at row 1

This part should be solved, feel free to reopen/open a new issue if you still run into errors.

Greets,
Jonas
